src/test/unit_tests/test_os_utils.js

```diff
@@ -0,0 +1,44 @@
+/* Copyright (C) 2016 NooBaa */
```

Comment:
I think that we should write the tests using `jest` and not `mocha`
@dannyzaken your input?